polaris-service/src/main/java/org/apache/polaris/service/config/PolarisApplicationConfig.java

```diff
@@ -193,15 +348,15 @@ public long getMaxRequestBodyBytes() {
     return maxRequestBodyBytes;
   }
 
-  public PolarisConfigurationStore getConfigurationStore() {
+  private PolarisConfigurationStore getConfigurationStore() {
```

Comment:
Note: this is not a getter, but creates a new instance.

Reply:
yeah, I'll rename to be more clear

However, the factory is registered as a singleton, so the factory method is only invoked once